### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by MadhurimaNayak** [ChromeGaming/Dot-Box#471]
### Give a brief about the bug ✍️

The performance of the site on mobile and the accessibility and SEO of the site on both mobile and desktop can be improved. 

### What is the expected behavior? 🤔

The performance, accessibility and SEO should be at least above 90. This enhances the user experience and makes the site more easy to be found. These are the current conditions:

**Mobile Stats:**
![image](https://github.com/ChromeGaming/Dot-Box/assets/124715194/f5fe0000-5faf-4cce-bf8d-f805f4f9c032)
**Desktop Stats:**
![image](https://github.com/ChromeGaming/Dot-Box/assets/124715194/90185c0d-a75e-4bde-b48d-c3b28ce6031e)

### Provide step by step information reproduce the bug 📄

You can use google lighthouse to analyze this information.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by MadhurimaNayak** [comment]
hey @ayush-t02 could you assign the level?
